portfolio/src/main/webapp/script.js

```diff
@@ -43,14 +43,59 @@ function deleteData() {
     });
 }
 
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyDjhk-U7NK6nQwmqSScCuZvO3l3_Q88Kps';
+script.defer = true;
+script.async = true;
+
+// Attach your callback function to the `window` object
+window.initMap = function() {
+  // JS API is loaded and available
+};
+
+// Append the 'script' element to 'head'
+document.head.appendChild(script);
```

Comment:
this can be obviated by adding the < script > to the html file directly

Reply:
I have    <script src="script.js"></script> in my html file. did you mean this ore something else.

Reply:
I'm suggesting something like < script src = https://maps.googleapis.com/maps/api/js?key=AIzaSyDjhk-U7NK6nQwmqSScCuZvO3l3_Q88Kps' > (with the additional properties. So you can add another <  script > tag directly to the HTML so that it loads the maps javascript

you can add multiple script tags to an HTML file (and in fact it's pretty common). You can kind of think of it as setting up imports in Java

There might be some things you need to do as far as ordering, though (I also don't know if you need or want async or defer)

Reply:
solved